### PR TITLE
Freetext dialogue state

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

File: go/base/static/js/test/tests/campaign/dialogue/states/freetext.test.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+describe("go.campaign.dialogue.states.choice", function() {
```
Comment (hodgestar) [inline go/base/static/js/test/tests/campaign/dialogue/states/freetext.test.js]: Should read `.freetext` not `.choice`?

File: go/base/static/js/test/tests/campaign/dialogue/states/freetext.test.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        editMode;
+
+    beforeEach(function() {
+      state = diagram.states.get('state2');
+      editMode = state.modes.edit;
+      state.edit();
+    });
+
+    describe(".save", function() {
+      it("should update the choice state's model", function() {
```
Comment (hodgestar) [inline go/base/static/js/test/tests/campaign/dialogue/states/freetext.test.js]: Should also say `freetext` (and for the test below).

**Comment by hodgestar** [comment]
Other than fixing the few places where test docs still reference `choice`, :+1:.

**Comment by justinvdm** [comment]
@hodgestar Thanks for catching the badly named tests. I really need to be more careful with copy-pasting.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**